src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -136,6 +144,21 @@ public org.folio.des.domain.dto.Job upsertAndSendToKafka(org.folio.des.domain.dt
       result.setExitStatus(ExitStatus.UNKNOWN);
     }
 
+    Optional.ofNullable(jobDto.getExportTypeSpecificParameters()).ifPresent(
+        expTypeSpecificParams -> Optional.ofNullable(expTypeSpecificParams.getVendorEdiOrdersExportConfig()).ifPresent(ediExportConfigParams ->
+          Optional.ofNullable(ediExportConfigParams.getExportConfigId())
+            .ifPresent(configId -> {
+              try {
+                log.info("Looking config with id {}", configId.toString());
+                client.getConfigById(configId.toString());
+              } catch (NotFoundException e) {
+                log.info("config not found", configId.toString());
+                ediExportConfigParams.getEdiSchedule().getScheduleParameters().setSchedulePeriod(ScheduleParameters.SchedulePeriodEnum.NONE);
+                throw new NotFoundException(String.format(INTEGRATION_NOT_AVAILABLE, configId));
```

Comment:
@okolawole-ebsco : i have tested it and the error is been thrown effectively. 
Anyway the main purpose of this catch block is to break and disable the next schedule , so I guess its okay if the error in not bubbling up . 
Kindly suggest...

Reply:
you can omit the throwing of the exception if all you needed was to disable the next schedule with the line above this. let me know how you want to move forward so that i may resolve this comment since you have confirmed that the error is bubbling up.

Reply:
We need to break the current job execution and then disable the schedule for further execution.
i have added a unit test for this .Kindly review .thanks much.

Reply:
looks good.